ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -35,27 +37,24 @@ typeof define === 'function' && define.amd ? define(['exports'], factory) :
   function createSidebar(ibrRawData, parentElement) {
     const ibrData = InternalBuildingRepresentation.read(
         new Pbf(ibrRawData));
-    // for datafiles that have top level name is ""
-    if (ibrData.name === '') {
-      ibrData.name = 'ibrData.name';
-    }
+    const curIBR = new IBRObject( ibrData );
     const li = document.createElement('li'); // list element container
     parentElement.appendChild(li);
-    const rootSpan = createLabel('span', ibrData.name, li);
+    const rootSpan = createLabel('span', curIBR.getName(), li);
     rootSpan.setAttribute('class', 'arrow');
     li.appendChild(rootSpan);
     const ul = document.createElement('ul');
     ul.setAttribute('class', 'nested');
-    ul.setAttribute('id', ibrData.name);
+    ul.setAttribute('id', curIBR.getName());
     li.appendChild(ul);
     rootSpan.addEventListener('click', function() {
       rootSpan.parentElement.querySelector('.nested').classList
           .toggle('active');
       rootSpan.classList.toggle('expanded-arrow');
     });
     // root structure index 0
-    const curStructure = renderSingleIBRStructure(ibrData, 0);
-    drawSingleStructureSidebar(curStructure, ibrData.name);
+    const curStructure = renderSingleIBRStructure(curIBR, 0);
```

Comment:
nit: curIBR looks like it might contain ALL the nested data. my suggestion would be even more specific and only pass relevant data to renderSinglIBRStructure (in a later PR)